kernelci.org/content/en/docs/org/tsc/_index.md

```diff
@@ -65,6 +65,12 @@ Votes done during a live meeting require 50% of the TSC members to be voting
 and a majority among these voting members.  Votes done by email or any other
 deferred channel require a majority among all of the TSC members.
 
+Abstensions are not counted as votes.  Effectively, they are the same as not
+attending a meeting or not replying to an email.  For example, with 7 TSC
+members in a meeting, if 3 are abstaining, 3 vote "yes" and 1 votes "no", we
+have quorum as 4/7 are voting and a majority as 3/4 voted "yes" - and the
+motion is moved.
```

Comment:
We discussed it yesterday in the TSC monthly and it was brought up in a board meeting once when someone abstained.  I said in that case that if we took it into account, it would basically amount to a "no" since it's one less vote towards a majority of "yes" and I believe the outcome was that it shouldn't be counted as such.

I've searched around a bit, interestingly in the House of Commons this is done by voting both "yes" and "no".  And the majority is based on the votes cast, with a minimum for quorum.  So if you "passively abstain" it doesn't count as a vote, and if you want to "actively abstain" you vote twice.

Maybe examples would help clarify this a bit.  In the case I described in this PR, if we took abstentions into account we have 3 "yes" out of 7 "votes" and the outcome is "no" even though there was only one actual "no" vote.  Would people agree this is the intended outcome?  Either way, if more than half of the voters abstain then the vote is rejected as there's either not a majority or no quorum.  Maybe a GitHub poll would help.

I suppose we could have a TSC vote about this paragraph or whatever we want to propose eventually, it's also why I put it in a separate PR as it's open to interpretation (as per the TSC rules, making changes to the rules requires a vote).  Now let's just hope people don't abstain when voting in this particular case...